docs/self-managed/backup-restore/optimize-backup.md

```diff
@@ -0,0 +1,127 @@
+
+```json
+{
+  "scheduledSnapshots": [
+    "camunda_optimize_backup1_3.9.0_part_1_of_2",
+    "camunda_optimize_backup1_3.9.0_part_2_of_2"
+  ]
+}
+```
+
+## Check backup state API
+
+Please note that the backup API can be reached via the `/actuator` management port, by default this is `8092`.  
+As the backup is created asynchronously, the current state of the backup can be checked by calling the following endpoint:
+
+```
+GET actuator/backup/{backupId}
+```
+
+### Response
+
+| Code             | Description                                                                                 |
+| ---------------- | ------------------------------------------------------------------------------------------- |
+| 200 OK           | Backup state could be determined and is returned in the response body (see example below)   |
+| 404 Not Found    | Backup with given id does not exist.                                                        |
+| 500 Server Error | All other errors, e.g. issues communicating with Elasticsearch for snapshot state retrieval |
+
+### Example request
+
+```
+curl 'http://localhost:8092/actuator/backup/backup1'
+```
+
+### Example response
+
+```json
+{
+  "State": "COMPLETED"
+}
+```
+
+Possible states of the backup:
+
+- `COMPLETED`: The backup can be used for restoring data.
+- `IN_PROGRESS`: The backup process for this backup ID is still in progress.
+- `FAILED`: Something went wrong when creating this backup. To find our the exact problem, one can use [Elasticsearch Get snapshot status API](https://www.elastic.co/guide/en/elasticsearch/reference/current/get-snapshot-status-api.html) for each of the snapshots included in given backup.
+- `INCOMPATIBLE`: The backup is incompatible with current Elasticsearch version.
+- `INCOMPLETE`: The backup is incomplete (this could occur when backup process was interrupted or individual snapshots were deleted).
+
+## Restore backup
+
+There is no Optimize API to perform the backup restore, instead the standard [Elasticsearch restore snapshot API](https://www.elastic.co/guide/en/elasticsearch/reference/current/restore-snapshot-api.html) can be used.
+
+:::note
+Optimize must NOT be running while a backup is being restored.
+:::
+
+To restore an existing backup, all the snapshots this backup contains (as listed in the response of the [create backup API request](#example-response)) must be restored using the Elasticsearch API.
+
+To restore a given backup, the following steps must be performed:
+
+1. Stop Optimize.
+2. Make sure there are no Optimize indices are present in Elasticsearch (otherwise the restore process will fail).
+3. Iterate over all Elasticsearch snapshots included in the desired backup and restore them using the Elasticsearch restore snapshot API.
+4. Start Optimize.
+
+Example Elasticsearch request:
+
+```
+curl --request POST `http://localhost:9200/_snapshot/repository_name/camunda_optimize_backup1_3.9.0_part_1_of_2/_restore?wait_for_completion=true`
+```
```

Comment:
General comment about this file - Do we need to say that this feature is experimental? Or will that be included in the general doc rather than the Optimize one?

Reply:
I actually added a note at the top which I didn't push 🙃 , will include in the next commit